### PR TITLE
update studio

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

File: docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md [hunk 1]
```diff
@@ -49,14 +49,16 @@ Before you deploy RPM-based Studio, you must confirm that:
    When the screen returns the following message, it means that the PRM-based Studio has been successfully started.
 
    ```bash
-   Created symlink from /etc/systemd/system/multi-user.target.wants/nebula-graph-studio.service to /usr/lib/systemd/system/nebula-graph-studio.service.
+   Start installing Nebula Studio now...
```
Comment (cooper-lzy) [inline docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md:52]: add $ in command.

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md [hunk 2]
```diff
@@ -33,13 +37,13 @@ To create a graph space on the **Schema** page, follow these steps:
 
   In the **Equivalent to the following nGQL statement** panel, you can see the statement equivalent to the preceding settings.
    
-   ```bash
-   CREATE SPACE basketballplayer (partition_num = 100, replica_factor = 1, vid_type = FIXED_STRING(32)) COMMENT = "Statistics of basketball players"
-   ```
+    ```bash
```
Comment (cooper-lzy) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md:40]: typo

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-edge-type.md [hunk 3]
```diff
@@ -75,23 +77,25 @@ To edit an edge type on the **Schema** page, follow these steps:
 
    - To delete the TTL configuration: When the **Set TTL** panel is expanded, in the upper left corner of the panel, click the check box to delete the configuration.
 
-6. When the configuration is done, in the **Equivalent to the following nGQL statement** panel, you can see the equivalent `ALTER EDGE` statement.
+    !!! note
+
+        The problem of coexistence of TTL and index, see [TTL]((../../3.ngql-guide/8.clauses-and-options/ttl-options.md).
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-edge-type.md:82]:
```suggestion
        For information about the coexistence problem of TTL and index, see [TTL]((../../3.ngql-guide/8.clauses-and-options/ttl-options.md).
```

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-tag.md [hunk 4]
```diff
@@ -52,48 +59,49 @@ To edit a tag on the **Schema** page, follow these steps:
...
-2. In **Graph Space List**, find a graph space, and then click its name or the button ![Icon of setting](https://docs-cdn.nebula-graph.com.cn/nebula-studio-docs/st-ug-018.png "Settings") in the **Operations** column.
+2. In the **Graph Space List** page, find a graph space and then click its name or click **Schema** in the **Operations** column.
 
 3. In the **Current Graph Space** field, confirm the name of the graph space. If necessary, you can choose another name to change the graph space.
 
-4. Click the **Tag** tab, find a tag and then the button ![Icon of deletion](https://docs-cdn.nebula-graph.com.cn/nebula-studio-docs/st-ug-017.png) in the **Operations** column.
+4. Click the **Tag** tab, find an edge type and then click the button ![Icon of deletion](https://docs-cdn.nebula-graph.com.cn/figures/alert-delete.png) in the **Operations** column.
 
-5. CLick **OK**.
+5. Click **OK** to confirm in the pop-up dialog box.
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-tag.md:104]: what to confirm?

File: docs-2.0/nebula-studio/quick-start/st-ug-console.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Console
+
+Studio console interface as shown in following.
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/quick-start/st-ug-console.md:3]: the predicate is missing. xxx is as shown in the following figure.
It is better to say `as shown in the following xxx`, or `xx is shown as follows`

File: docs-2.0/nebula-studio/quick-start/st-ug-console.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+|  11  |  export CSV file or PNG file |  After running the nGQL statement to return the result, when the result is in **Table** window, click the ![download](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-download.png) button to export as a CSV file. Switch to the **Graph** window and click the ![download](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-download.png) button to save the results as a CSV file or PNG image export. |
+|  12  |  expand/hide execution results  | Click the ![up](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-up.png) button to hide the result or click ![down](https ://docs-cdn.nebula-graph.com.cn/figures/studio-btn-down.png) button to expand the result. |
+|  13  |  close execution results | Click the ![close](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-close.png) button to close the result returned by this nGQL statement. |
+|  14  |  **Table** window | Displays the result from running nGQL statement. If the statement returns results, the window displays the results in a table. |
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/quick-start/st-ug-console.md:24]: it's better to keep the consistency of singular and plural forms.


Clicks or Click xxxx
Displays or Display xxx

File: docs-2.0/nebula-studio/quick-start/st-ug-console.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+![console](https://docs-cdn.nebula-graph.com.cn/figures/st-ug-015-en.png)
+
+The following table lists various functions on the console interface.
+
+| number  |  function | descriptions  |
+| :-- | :--|   :--   |
+|  1  |  toolbar  |  Click the **Console** tab to enter the console page.  |
+|  2  |  select a space  | Select a space in the Current Graph Space list. <br/> **descriptions**: Studio does not support running the `USE <space_name>` statements directly in the input box.  |
+|  3  | favorites | Click the ![save](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-save.png) button to expand the favorites, click one of the statements, and the input box will automatically enter the statement. |
+|  4  |  history list   |  lick ![history](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-history.png) button representing the statement record. In the statement running record list, click one of the statements, and the statement will be automatically entered in the input box. The list provides the record of the last 15 statements.  |
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/quick-start/st-ug-console.md:14]: lick ->Click?

File: docs-2.0/nebula-studio/quick-start/st-ug-console.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+|  12  |  expand/hide execution results  | Click the ![up](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-up.png) button to hide the result or click ![down](https ://docs-cdn.nebula-graph.com.cn/figures/studio-btn-down.png) button to expand the result. |
+|  13  |  close execution results | Click the ![close](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-close.png) button to close the result returned by this nGQL statement. |
+|  14  |  **Table** window | Displays the result from running nGQL statement. If the statement returns results, the window displays the results in a table. |
+|  15  |  **Graph** window | Displays the result from running nGQL statement. If the statement returns the complete vertex-edge result, the window displays the result as a graph . Click the ![expand](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-back.png) button on the right to view the overview panel. |
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/quick-start/st-ug-console.md:25]: ditto

**Review by cooper-lzy — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**